vault-upgrade-hook/src/main/java/biz/netcentric/vlt/upgrade/handler/UpgradeActionInfo.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (C) Copyright 2016 Netcentric AG.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package biz.netcentric.vlt.upgrade.handler;
+
+import javax.jcr.Node;
+
+import org.apache.jackrabbit.vault.packaging.InstallContext.Phase;
+
+public class UpgradeActionInfo {
```

Comment:
Do we really need this additional class? It looks like kind of a subset of ActionInfo and makes the whole class structure even more confusing as it already is.

Reply:
Transfering a simple DTO from UpgradeInfo to handler classes decreases coupling between those classes.

Also, most of the data inside UpgradeInfo is just not needed in the handlers. 

And, as I wrote in the comment to the PR, because of some kind of circular dependency (create param for the method under test -> parameter's constructor call method under the test) makes testing without many mocks quite hard.

Reply:
Decision: Call loadActions() not in the constructor of UpgradeInfo but separately.